src/rustllvm/RustWrapper.cpp

```diff
@@ -1395,3 +1395,49 @@ LLVMRustModuleCost(LLVMModuleRef M) {
   auto f = unwrap(M)->functions();
   return std::distance(std::begin(f), std::end(f));
 }
+
+// Vector reductions:
+extern "C" LLVMValueRef
+LLVMRustBuildVectorReduceFAdd(LLVMBuilderRef B, LLVMValueRef Acc, LLVMValueRef Src) {
+    return wrap(unwrap(B)->CreateFAddReduce(unwrap(Acc),unwrap(Src)));
```

Comment:
I think CI may be failing on these in the sense that they're not defined on LLVM 3.9, but it's fine to have some `#ifdef` here to only compile these in on 6.0+ and otherwise return errors on older versions